run.py

```diff
@@ -1,28 +1,31 @@
 import argparse
-import logging
-import yaml
 import glob
+import logging
 import os
-from EOS.helper import run_eos
+import sys
+import yaml
 
+from tests.storage.helper import run_eos
 
 def get_args():
     parser = argparse.ArgumentParser(description='Arguments', formatter_class = argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument('-t', '--test',
                         nargs='*', default='compTest',
                         required=False,
                         help='name of the test you want to run')
-    parser.add_argument('-j', '--json',
-                        required=False,
-                        action='store',
-                        help='Output to json file')
+    parser.add_argument('-c', '--configfile',
+                        required = True,
+                        help='load config file')
     args = parser.parse_args()
     return args
 
 
 def get_config(cfg):
-    with open(cfg, 'r') as stream:
-        return yaml.safe_load(stream)
+    if os.path.exists(cfg):
+        with open(cfg, 'r') as stream:
+         return yaml.safe_load(stream)
+    else:
```

Comment:
Perfect!
Consider throwing a message to the user if the file is not there

Reply:
Done!